demo/auth/passport.js

```diff
@@ -0,0 +1,47 @@
+var $fh = require('fh-js-sdk');
+var logger = require('@raincatcher/logger').logger;
+var ConsoleLogger = require('@raincatcher/logger').ConsoleLogger;
+var setLogger = require('@raincatcher/logger').setLogger;
+
+$fh.init({}, function() {
+  var passport = {
+    loadUserProfile: function($http, $window) {
+      var req = {
+        method: 'GET',
+        url: $fh.getCloudURL() + '/profile'
+      };
+      return $http(req, {withCredentials: true}).then(function (res) {
+        return res.data;
```

Comment:
We should store user data in local storage to follow the same approach as keycloak does and provide offline capabilities ( as old solution ). Doing it this way will produce a lots of bugs..

Note: This will also require additional handling in the logout code.